source/pkcore/abilitypages/chi/swiftkick.rst

```diff
@@ -0,0 +1,63 @@
+.. swiftkick:
+############
+SwiftKick
+############
+
+Description
+###########
+
+SwiftKick is a Chi ability.
+
+SwiftKick allows a chiblocker to swiftly kick an enemy, potentially blocking their chi. GIF TO BE ADDED
+
+
+Usage
+#####
+
+USAGE GIF TO BE ADDED 
+
+Function 1: [Left Click] the target entity.
```

Comment:
This is missing a key component of this abilities usage. The user must be midair while they click for the move to activate. See https://github.com/ProjectKorra/ProjectKorra/blob/master/src/com/projectkorra/projectkorra/chiblocking/SwiftKick.java#L40-L43